web/html/src/manager/systems/virtual-list.tsx

```diff
@@ -70,7 +68,7 @@ export function VirtualSystems(props: Props) {
         <Column
           columnKey="name"
           comparator={Utils.sortByText}
-          header={t("Virtual System")}
+          header={t("Virtual Machine")}
```

Comment:
Better not change this name: the word `System` is used all over the place in the user interface and docs to refer to a machine. But this is nitpicking

Reply:
Got you! Updated the label on the dropdown as well.